Jenkinsfile

```diff
@@ -71,4 +102,20 @@ pipeline {
             }
         }
     }
-}
+
+    post {
```

Comment:
cleanup parallel to server pipeline to cleanup spawned repose from testing and save space